### PR TITLE
implement header, staple, and gpt icons

### LINKED ISSUE #32: Buttons & Icons
I've created space for some icons for additional functionality. At the moment, I'd like for these icons to only be clickable without any additional functionality for now.
Icons should have the ability to animate when clicked as well as adjust to theme.

Untitled design (16)

### PR DESCRIPTION
**Pull request by rosaleensiroosi**
resolves #32.

### CODE REVIEW

File: MapGPT/components/header.tsx [hunk 1]
```diff
@@ -13,12 +15,32 @@ export const Header: React.FC = async () => {
           <span className="sr-only">Chat</span>
         </a>
       </div>
-      <div className="flex gap-0.5">
+      <div className="w-1/2 md:p-2 justify-between px-6">
+        <Button variant="ghost" size="icon">
+          <img src="/images/logo.png" alt="Logo" className="h-6" />
```
Comment (ngoiyaeric) [inline MapGPT/components/header.tsx:20]: This image should be dynamic, try converting it to an SVG.

**Review by ngoiyaeric — CHANGES_REQUESTED** [review]
add a screenshot so I can see the positioning and size for the icons.

**Comment by rosaleensiroosi** [comment]
image

File: MapGPT/components/header.tsx (hunk 1 above)
Comment (ngoiyaeric) [reply MapGPT/components/header.tsx:20]: @rosaleensiroosi https://chatgpt.com/share/1ed13f02-8738-41a2-a8da-76e8eb5a7e06

**Comment by ngoiyaeric** [comment]
I've also noticed that the logo disappears when there are chats to scroll, its best to have it all the way to the left of the screen and not above the chat. Also if you could arrange the order of the icons as such. 
Screen Shot 2024-06-11 at 11 14 38 AM

**Pull request merged**